open_academy/views/session_views.xml

```diff
@@ -32,10 +33,58 @@
             </form>
         </field>
     </record>
+    <record id="session_view_calendar" model="ir.ui.view">
+        <field name="name">session.calendar</field>
+        <field name="model">session</field>
+        <field name="arch" type="xml">
+            <calendar string="Events" date_start="start_date">
+                <field name="name"/>
+            </calendar>
+        </field>
+    </record>
+    <record id="session_view_gantt"  model="ir.ui.view" >
+        <field name="name">session.view.gantt</field>
+        <field name="model">session</field>
+        <field name="arch" type="xml">
+            <gantt 
+                string="Gantt"
+                date_start="start_date"
+                date_stop="end_date"
+                default_group_by="instructor_id" 
+                default_scale = "week"
```

Comment:
are this spaces intended?

Reply:
why?

Reply:
It's just for a better readability , due to the gantt tag has a lot of attributes

Reply:
I've meant this spaces between the '=' here `default_scale = "week"`

Are there like that for a reason?

If not, could you check please if `default_scale="week"` keeps working?